### DISCUSSION TITLE
ProjectInto not working with property named Order

### DISCUSSION THREAD

**Discussion by nant1086** [ravendb/ravendb#15171] [Q&A]
Hi all, I just found an issue while trying to use ProjectInto&lt;T&gt;(), I have a property named Order (sales order) but when I execute the ProjectInto&lt;T&gt;() method, I just get the property name instead
``` JSON
{
  "Id": "ServiceOrder/4201-A",
  "Order": "Order",
  "Customer": "customer/45-A"
}
```
while using the Query&lt;T&gt;().ToList() I got the correct value for Order property
``` JSON
{
  "Id": "ServiceOrder/4201-A",
  "Order": "SalesOrder/121723-A",
  "Customer": "customer/45-A",
  "@metadata": {
     "@collection": "ServiceOrder"
  }
}
```
Is there a workaround or plan to fix this?

**Comment by garayx** [comment]
Hi 

Can you send a sample code which reproduces this ? (see: https://ravendb.net/docs/article-page/5.1/Csharp/start/test-driver)

**Comment by grisha-kotler** [comment]
You can track the status of this issue here:
https://issues.hibernatingrhinos.com/issue/RavenDB-19482/Projection-with-reserved-keywords-doesnt-work
